src/componentSetBuilder.ts

```diff
@@ -82,10 +82,10 @@ export class ComponentSetBuilder {
         metadata.metadataEntries.forEach((rawEntry) => {
           const splitEntry = rawEntry.split(':');
```

Comment:
```suggestion
         const splitEntry = rawEntry.split(':').map(entry => entry.trim())
```

will trim both, so you don't have to trim in 2 places OR worry about the untrimmed type on line 87